File: pset6/pset6/mario/more/mario.py
from cs50 import get_int

height = get_int("Height: ")  # Takes the input of the user.

if height > 8 or height < 1:
    while (height > 8) or (height < 1):  # If the input is greater than 8 or less than 1, a loop will run until required input is given.
        height = get_int("Height: ") 
    
height = height
m = height
for i in range(height):  # For printing rows
    for l in range(0, m-1):  # For printing blanks
        print(" ", end="")
    
    for j in range(0, i+1):  # For printing first pyramid of hashes.
        print("#", end="")
        
    print(" ", end=" ")  # For putting spaces in between
        
    for k in range(0, i+1):  # For printing the second pyramid of hashes.
        print("#", end="")
    print("")
    m = m - 1

File: pset1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    //To take an input of a number between 1 and 8
    do
    {
        height = get_int("Height :");
    }
    while (height > 8 || height < 1);
    //To print the required pyramid
    for (int i = height; i >= 1; i--)
    {
        for (int j = 1; j <= height; j++)
        {
            if (j >= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }
        printf(" ");
        for (int j = i; j >= height ; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}    